src/main/res/layout/statistics_recording.xml

```diff
@@ -2,7 +2,6 @@
 <androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
-    style="@style/StatsScrollView"
```

Comment:
Why?

Reply:
We don't need it because the scroll bar will be containted into the RecyclerView but not in this ConstraintLayout.